### PR TITLE
Follow-up edits to PR #2150

### PR DESCRIPTION
**Pull request by tnguyen-rh**
Follow-up edits to PR #2150 

@adellape @ahardin-rh @tpoitras @bfallonf 
Peer review please?

### CODE REVIEW

**Comment by tnguyen-rh** [comment]
[rev_history]
|link:../dev_guide/deployments.html[Deployments]
|Clarified operational conditions around link:../dev_guide/deployments.html#config-change-trigger[config-change] and link:../dev_guide/deployments.html#image-change-trigger[image-change] triggers.
%

File: dev_guide/deployments.adoc [hunk 1]
```diff
@@ -275,8 +274,8 @@ specified in the deployment configuration's `helloworld` container, a new
 deployment is created using the new image for the `helloworld` container.
 
 NOTE: If an ImageChange trigger is defined on a deployment configuration (with
-a ConfigChange trigger or with automatic=true) and the ImageStreamTag pointed
-by the ImageChange trigger does not exist yet then the first deployment will
+a ConfigChange trigger or with `automatic=true`) and the ImageStreamTag pointed
+by the ImageChange trigger does not exist yet, then the first deployment will
```
Comment (bfallonf) [inline dev_guide/deployments.adoc]: @tnguyen-rh I know this was there already, but I'd get rid of the "will":

"...then the first deployment automatically starts as soon as..."

But also, does ImageStreamTag and ImageChange need markup?

**Comment by bfallonf** [comment]
@tnguyen-rh Nits about stuff that was already there, but LG otherwise :deciduous_tree:

**Comment by tnguyen-rh** [comment]
@bfallonf 
Thanks for the review.  I eliminated the "will" as you suggested, and added markup, too.

**Pull request merged**